internal/commands/app/describe_test.go

```diff
@@ -0,0 +1,380 @@
+package app
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/atlas"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/local"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+
+	"github.com/Netflix/go-expect"
+)
+
+func TestDescribeInputsResolve(t *testing.T) {
```

Comment:
[nit] I probably havent done a great job enforcing this, but best to name these tests in a cascading fashion so the regex targeting can be a bit more effective: `TestAppDescribe...`